be/src/olap/memtable.cpp

```diff
@@ -64,8 +64,8 @@ MemTable::MemTable(int64_t tablet_id, Schema* schema, const TabletSchema* tablet
         } else {
             _insert_fn = &MemTable::_insert_agg;
         }
-        if (_tablet_schema->has_sequence_col()) {
-            _aggregate_two_row_fn = &MemTable::_aggregate_two_row_with_sequence;
```

Comment:
Is sequence_col in tablet_schema not needed？what abort old meta info？

Reply:
This part of code is not removed, just move to `_aggregate_two_row`, it works quite similar like the method `_aggregate_two_row_in_block`.
For the unique table without MoW(merge-on-write), they all proceeded by the method `_aggregate_two_row`, no necessary to separate to 2 methods.